### PR TITLE
Tpt3 77

### PR DESCRIPTION
**Pull request by zemlyanoi**

### CODE REVIEW

**Comment by zemlyanoi** [comment]
MERGE AFTER API-CORE
https://github.com/ca-cwds/api-core/pull/57

File: src/main/java/gov/ca/cwds/cm/service/dictionaries/AssignmentType.java [hunk 1]
```diff
@@ -4,7 +4,25 @@
  * @author CWDS TPT-3 Team
  */
 public enum AssignmentType {
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/cm/service/dictionaries/AssignmentType.java]: create this enum in api-core, use it in entity mapping and reuse it

File: src/main/java/gov/ca/cwds/cm/service/dto/ReferralDTO.java [hunk 2]
```diff
@@ -6,17 +6,19 @@
...
 import javax.validation.constraints.Size;
 import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 /** @author CWDS TPT-3 Team */
 @Data
 @NoArgsConstructor
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/cm/service/dto/ReferralDTO.java]: @NoArgsConstructor may be removed

File: src/main/java/gov/ca/cwds/cm/service/dto/facade/CaseByStaff.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author CWDS TPT-3 Team
+ */
+@Data
+@EqualsAndHashCode(callSuper = false)
+@NoArgsConstructor
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/cm/service/dto/facade/CaseByStaff.java]: may @NoArgsConstructor and @AllArgsConstructor be removed?

File: src/main/java/gov/ca/cwds/cm/service/facade/CaseLoadFacade.java [hunk 4]
```diff
@@ -1,33 +1,57 @@
...
-    return Arrays.asList(getMockedData());
+  private AssignmentDao assignmentDao;
+  private ReferralService referralService;
+  private CaseService caseService;
+
+  @Inject
+  public CaseLoadFacade(
+      AssignmentDao assignmentDao, ReferralService referralService, CaseService caseService) {
+    this.assignmentDao = assignmentDao;
+    this.referralService = referralService;
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/cm/service/facade/CaseLoadFacade.java]: caseService is missed

**Pull request merged**